apollo-client/src/main/java/com/ctrip/framework/apollo/Config.java

```diff
@@ -18,7 +18,7 @@
    * @param defaultValue the default value when key is not found or any error occurred
    * @return the property value
    */
-  public String getProperty(String key, String defaultValue);
```

Comment:
I would like to keep those unnecessary `public` modifiers, they are redundant but serve a clear sign for users.